spec/functions/join_keys_to_values_spec.rb

```diff
@@ -10,7 +10,12 @@
   it { is_expected.to run.with_params({}, '').and_return([]) }
   it { is_expected.to run.with_params({}, ':').and_return([]) }
   it { is_expected.to run.with_params({ 'key' => 'value' }, '').and_return(['keyvalue']) }
+  it { is_expected.to run.with_params({ 'ҝẽγ' => '√ạĺűē' }, '').and_return(['ҝẽγ√ạĺűē']) }
   it { is_expected.to run.with_params({ 'key' => 'value' }, ':').and_return(['key:value']) }
+  it { is_expected.to run.with_params({ 'ҝẽγ' => '√ạĺűē' }, ':').and_return(['ҝẽγ:√ạĺűē']) }
+  it { is_expected.to run.with_params({ 'ҝẽγ' => '√ạĺűē' }, '万').and_return(['ҝẽγ万√ạĺűē']) }
+
```

Comment:
I have now updated the context for the above tests to make it more clear what the test is doing and its purpose.